src/components/DataDrawer/index.tsx

```diff
@@ -32,7 +32,7 @@ const styles = (theme: Theme) =>
       backgroundColor: theme.palette.primary.main,
       padding: 16,
       width: '40vw',
-      height: '100vh',
+      height: '200vh',
```

Comment:
This is saying to make it twice as tall as the screen, is that really want you want here? Or is the idea to make the drawer as tall as the screen and then have the chart take up half the vertical space?